### PR TITLE
New boss + Updated Russian language

### PR DESCRIPTION
**Pull request by Blueberryy**
New boss "Night Stalker"
12 Damage
850 Health

### CODE REVIEW

File: entities/weapons/weapon_zs_fastzombie_hunter/shared.lua [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+SWEP.IsPouncing = SWEP.GetPouncing

+

+if CLIENT then return end

+

+function SWEP:Deploy()
+	self.Owner:CreateAmbience("fastzombieambience")
```
Comment (MrCraigTunstall) [inline entities/weapons/weapon_zs_fastzombie_hunter/shared.lua]: where is the "fastzombieambience" ambience? Also please fix the linings so there is no "^M"

Comment (Blueberryy) [reply entities/weapons/weapon_zs_fastzombie_hunter/shared.lua]: fastzombieambience already in gamemode https://github.com/MrCraigTunstall/zombiesurvival/tree/master/entities/entities/status_fastzombieambience

**Comment by Blueberryy** [comment]
I don't know what's wrong but when i'm uploading on github it makes these "^M"

**Comment by Blueberryy** [comment]
But i can't see "^M" in lua files

**Comment by Blueberryy** [comment]
notepad++_2gCIa0KNIs
notepad++_nKZ5nUIAqu

Strange...

**Pull request merged**